MSAL/src/configuration/external/ios/MSALLegacySharedAccount.m

```diff
@@ -31,7 +31,9 @@ @interface MSALLegacySharedAccount()
 
 @property (nonatomic, readwrite) NSDictionary *jsonDictionary;
 @property (nonatomic, readwrite) NSString *username;
-
+@property (nonatomic, readwrite) NSArray<MSALTenantProfile *> *tenantProfiles;
```

Comment:
why do we need to declare these properties in legacy account? It doesn't implement MSALAccount protocol.

Reply:
Updated